lib/rbs/inline/ast/members.rb

```diff
@@ -376,11 +378,7 @@ def rbs
             return unless node.arguments.arguments.size == 1
 
             arg = node.arguments.arguments[0] || raise
-            if arg.is_a?(Prism::ConstantReadNode)
-              type_name = RBS::TypeName.new(name: arg.name, namespace: RBS::Namespace.empty)
-            else
-              raise
-            end
+            type_name = type_name(arg) || raise
```

Comment:
Don't raise in the case of unexpected mixin argument. It skips processing the mixin and continue parsing.

```rb
class Foo
  include (x = Bar)   # Should continue processing for next `include`
  include Baz
end
```

Reply:
I fixed it so that it continues parsing in the case where the mixin argument is a dynamic value.
I've also added a test for this case.